src/main/java/com/testsigma/sdk/Email.java

```diff
@@ -14,6 +16,8 @@ public interface Email {
   void setAttachments(List<File> attachments);
 
   void setContentType(ContentType contentType);
+  void addHeader(Map<String, String> headersList);
 
+  String getMessageId(String headers);
```

Comment:
@navyana-srivalli is it required over here?

Reply:
To get message ID from headers i added it over here. 
It will be the common method if we have more NLPs in future.